### PR TITLE
Minion content staging

### PR DESCRIPTION
**Pull request by mbologna**
Travis test fail due to:
./features/cve_audit.feature:53: Remove 16 empty lines

which was not part of this PR.

### CODE REVIEW

**Comment by MalloZup** [comment]
The rubocop error got fixed , git.pull

**Comment by mbologna** [comment]
@MalloZup rebased on master. All checks passed. Thanks!

**Comment by MalloZup** [comment]
@mbologna  for me it's ok! 
@Bischoff if you have nothing against it i will merge it today

File: features/salt_install_with_staging.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    And I wait until the package "orion-dummy-1.1-1.1.x86_64" has been cached on this "sle-minion"
+    And I wait for "orion-dummy-1.1-1.1" to be installed on this "sle-minion"
+    Then I remove pkg "orion-dummy-1.1-1.1" on this "sle-minion"
+
+   Scenario: Install patch in the future and check for staging
+    Given I am on the Systems overview page of this "sle-minion"
+    And I remove pkg "virgo-dummy" on this "sle-minion"
+    And I enable repository "Devel_Galaxy_BuildRepo" on this "sle-minion"
+    And I install pkg "virgo-dummy-1.0-1.1" on this "sle-minion"
+    And I wait for "30" seconds
```
Comment (MalloZup) [inline features/salt_install_with_staging.feature]: the only weak-point maybe it's only this wait 30 seconds. why you are waiting for 30 sec? can we just do and wait virgo-mummy to be installed on sles-bingo-bongo? i think we have a function for that

File: features/salt_install_with_staging.feature [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    And I follow "Software" in the content area
+    And I click on "Update Package List"
+    And I wait for "30" seconds
```
Comment (MalloZup) [inline features/salt_install_with_staging.feature]: this 30 seconds it's only for waiting for refresh?

**Comment by MalloZup** [comment]
@mbologna  a part for the 2 comments, it looks pretty nice your pr! thx for that 🥇

File: features/salt_install_with_staging.feature (hunk 1 above)
Comment (MalloZup) [inline features/salt_install_with_staging.feature]: https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/salt_install_package.feature#L18

Comment (mbologna) [inline features/salt_install_with_staging.feature]: "wait to be installed" steps only checks and waits until the package has been installed (aka we can't use it to install a package):

https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/step_definitions/salt_steps.rb#L593-L612

In this case we need to:

* install a package via zypper
* wait the zypper beacon to check-in with Salt about the installed package

That's why we are using "install pkg" and waiting for 30 seconds.

File: features/salt_install_with_staging.feature (hunk 2 above)
Comment (mbologna) [inline features/salt_install_with_staging.feature]: Yes, waiting for the zypper beacon to check-in with Salt about the installed package

**Pull request merged**